Setup php-cs-fixer, refs #13317

jraddaoui opened the pull request:
Format PHP code using PHP CS Fixer and the @PhpCsFixer ruleset with
one exception to avoid inconsitencies in templates (check `.php_cs`).
Setup Github Actions workflow to run syntax checks in the CI process.

jraddaoui commented on plugins/sfThumbnailPlugin/lib/sfImageMagickAdapter.class.php:
```diff
@@ -340,7 +340,7 @@ public function save($thumbnail, $thumbDest, $targetMime = null)
           $this->image = $thumbDest;
         }
-      	break;
+          break;
```
Indentation fixed later.

jraddaoui commented on apps/qubit/modules/default/templates/_breadcrumb.php:
```diff
@@ -2,7 +2,7 @@
   <ul>
     <?php foreach ($objects as $object): ?>
-      <?php if (isset($object->parent)): // FIXME Implement something like ->slice(1) or [1:] ?>
+      <?php if (isset($object->parent)): // FIXME Implement something like ->slice(1) or [1:]?>
```
Missing whitespace before `?>` is fixed later.

jraddaoui commented on apps/qubit/modules/actor/actions/browseAction.class.php:
```diff
@@ -39,8 +39,8 @@ class ActorBrowseAction extends DefaultBrowseAction
-    ),
-
+    );
+  public static
```
Variable name is moved to the same line in a later commit.

jraddaoui commented on apps/qubit/modules/actor/actions/browseAction.class.php:
```diff
@@ -31,54 +31,54 @@ class ActorBrowseAction extends DefaultBrowseAction
   // Arrays not allowed in class constants
-  public static $NAMES = array(
+  public static $NAMES = [
       'repository',
```
Indentation is fixed later.

jraddaoui commented on apps/qubit/modules/clipboard/actions/saveAction.class.php:
```diff
@@ -63,7 +63,7 @@ private function getUniquePassword()
     // Try a max of 100 times before giving up (avoid infinite loops when
     // possible passwords exhausted)
-    for ($i = 0; $i < 100; $i++)
+    for ($i = 0; $i < 100; ++$i)
```
I thought this would change the for behavior (adding before starting the count), but it doesn't.

jraddaoui commented on lib/Qubit.class.php:
I have doubts about this changes ... specially in the regular expressions ...

https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.18/doc/rules/string_notation/escape_implicit_backslashes.rst

jraddaoui replied:
I think it's okay in the end @djjuhasz ...

https://stackoverflow.com/a/23215075
https://www.php.net/manual/en/regexp.reference.escape.php

Render date method seems to be working okay and the model builder changes don't introduce changes in the actual models when they're built.

jraddaoui commented on apps/qubit/modules/actor/templates/_occupations.php:
```diff
@@ -11,7 +11,7 @@
 
     <?php $i = 0; foreach ($occupations as $item) { ?>
-      <?php $form->getWidgetSchema()->setNameFormat("occupations[$i][%s]"); ?>
+      <?php $form->getWidgetSchema()->setNameFormat("occupations[${i}][%s]"); ?>
```
These are normalized later to `{$i}`, like the method/property calls (`{$request->query}` above).

jraddaoui commented on apps/qubit/modules/clipboard/actions/exportAction.class.php:
```diff
@@ -516,23 +516,17 @@ private function getJobNameString()
-        else
-        {
+
           return 'arInformationObjectXmlExportJob';
```
Broken indentation from this commit is fixed later.

djjuhasz replied:
🤯 😆

jraddaoui commented on plugins/sfEadPlugin/modules/sfEadPlugin/templates/indexSuccessBody.xml.php:
```diff
@@ -464,7 +463,7 @@
     <?php } ?>
 
-    <?php // Make sure all <c> tags are closed ?>
+    <?php // Make sure all <c> tags are closed?>
```
Remove this one

djjuhasz commented on apps/qubit/modules/clipboard/actions/exportAction.class.php:
```diff
@@ -340,7 +340,6 @@ protected function addField($name)
         );
 
         break;
-
```
No blank line between `break;` and the comment is a bit weird 😕

djjuhasz commented on apps/qubit/modules/actor/actions/browseAction.class.php:
```diff
@@ -32,53 +32,61 @@ class ActorBrowseAction extends DefaultBrowseAction
-        ['type' => 'term',
-              'field' => 'subjects.id',
-              'size' => 10]];
+    'languages' => [
```
*So much better* 👏

djjuhasz commented on docker/bootstrap.php:
```diff
@@ -59,25 +59,25 @@ function get_host_and_port($value, $default_port)
-# /apps/qubit/config/settings.yml
-#
+//
+// /apps/qubit/config/settings.yml
```
Eww. 😬

djjuhasz commented on apps/qubit/modules/actor/templates/_advancedSearch.php:
```diff
@@ -55,7 +55,7 @@
           </select>
 
-          <input class="query" type="text" placeholder="<?php echo __('Search'); ?>" name="sq<?php echo $count;?>"/>
+          <input class="query" type="text" placeholder="<?php echo __('Search'); ?>" name="sq<?php echo $count; ?>"/>
```
🙌

djjuhasz commented on apps/qubit/modules/actor/actions/browseAction.class.php:
```diff
@@ -35,7 +35,7 @@ class ActorBrowseAction extends DefaultBrowseAction
     'emptyField',
     'relatedType',
-    'relatedAuthority'
+    'relatedAuthority',
```
🙌

djjuhasz commented on apps/qubit/modules/default/templates/_genericPicker.php:
```diff
@@ -8,7 +8,7 @@
         <?php $options = $sf_data->getRaw('options'); ?>
         <?php $param = $sf_data->getRaw('param'); ?>
-        <?php if (isset($sf_request->$param) && isset($options[$sf_request->$param])) { ?>
+        <?php if (isset($sf_request->$param, $options[$sf_request->$param])) { ?>
```
Wow, I didn't even know you *could* combine "isset" statements in PHP 🤯

jraddaoui replied:
Yeah, me neither :grin:

djjuhasz commented on apps/qubit/modules/user/actions/loginAction.class.php:
```diff
@@ -88,8 +88,7 @@ public function execute($request)
       // and email to them when the login does not succeed.
-      unset($request['email']);
-      unset($request['password']);
+      unset($request['email'], $request['password']);
```
🤯 x 2

djjuhasz commented on apps/qubit/modules/actor/actions/browseAction.class.php:
```diff
@@ -339,7 +339,7 @@ protected function addField($name, $request)
         if (!empty($this->relatedAuthority))
         {
-          $defaultChoices = [$request->$name => $this->relatedAuthority->getAuthorizedFormOfName(['cultureFallback' => true])];
+          $defaultChoices = [$request->{$name} => $this->relatedAuthority->getAuthorizedFormOfName(['cultureFallback' => true])];
```
👍

djjuhasz commented on lib/PhysicalobjectCsvImporter.class.php:
```diff
@@ -278,7 +278,7 @@ public function setHeader(string $str = null)
     if (empty($columnNames))
     {
-      $msg = <<<EOM
+      $msg = <<<'EOM'
```
🤔 👍

djjuhasz commented on apps/qubit/modules/default/actions/fullTreeViewAction.class.php:
```diff
@@ -180,7 +180,8 @@ protected function doElasticsearchQuery($term, $options = [])
       ->getType('QubitInformationObject')
-      ->search($query->getQuery(false, false));
+      ->search($query->getQuery(false, false))
+    ;
```
🤯 👍

djjuhasz commented on apps/qubit/modules/search/actions/globalReplaceAction.class.php:
```diff
@@ -37,7 +37,7 @@ public function execute($request)
       }
       // Make sure we have confirmed the action
-      elseif (!isset($request->confirm))
+      if (!isset($request->confirm))
```
Haha, I think php-cs-fixer is smarter than I am 😝

jraddaoui replied:
hahaha

djjuhasz commented on apps/qubit/config/qubitConfiguration.class.php:
```diff
@@ -143,8 +143,8 @@ public static function getConfigForEnvironment($varname, $env, $configFile)
     if (
-      'config/settings.yml' == $configFile &&
-      isset($config[$env]['.settings'][$varname]))
+      'config/settings.yml' == $configFile
```
👍

djjuhasz commented on test/phpunit/PhysicalObjectCsvHoldingsReportTest.php:
```diff
@@ -2,6 +2,7 @@
 
 /**
  * @internal
+ * @coversNothing
```
I think this should be

```
* @covers PhysicalObjectCsvHoldingsReport
```

djjuhasz commented on test/phpunit/PhysicalobjectCsvImporterTest.php:
```diff
@@ -4,6 +4,7 @@
 
 /**
  * @internal
+ * @coversNothing
```
I think this should be:

```
* @covers PhysicalObjectCsvImporter
```

djjuhasz commented on apps/qubit/config/qubitConfiguration.class.php:
```diff
@@ -107,9 +107,7 @@ public function getTemplateDirs($moduleName)
-    $dirs = array_merge($dirs, $this->getDecoratorDirs());
-
-    return $dirs;
+    return array_merge($dirs, $this->getDecoratorDirs());
```
😮 👏

djjuhasz commented on apps/qubit/config/qubitConfiguration.class.php:
```diff
@@ -51,8 +51,7 @@ public function configure()
   {
-    if (false !== $readOnly = getenv('ATOM_READ_ONLY'))
-    {
+    if (false !== $readOnly = getenv('ATOM_READ_ONLY')) {
```
🙌

djjuhasz commented on plugins/sfInstallPlugin/modules/sfInstallPlugin/actions/taskComponent.class.php:
```diff
@@ -21,5 +21,56 @@ class sfInstallPluginTaskComponent extends sfAction
 {
     public function execute($request)
     {
+        $this->checkSystemStatus;
```
🙇

djjuhasz commented on apps/qubit/modules/search/templates/_aggregation.php:
```diff
@@ -1,11 +1,10 @@
-<section class="facet <?php if ($openned) {
-    echo 'open';
-} ?>">
+<section class="facet <?php echo $openned ? 'open' : ''; ?>">
```
😮 👏

djjuhasz reviewed (changes requested):
Wow, looks great @jraddaoui!  👏 👏 👏  

I only had one requested change about the PHP Unit `@covers` annotations.

jraddaoui commented:
Thanks @djjuhasz, done! I'd appreciate your feedback too in the wiki changes I added to the proposal doc.

djjuhasz reviewed (approved):
Great job @jraddaoui!

jraddaoui committed:
Setup php-cs-fixer, refs #13317

Format PHP code using PHP CS Fixer and the @PhpCsFixer ruleset with
one exception to avoid inconsitencies in templates (check .php_cs).
Setup Github Actions workflow to run syntax checks in the CI process.

Squashed commit of the following:

commit 6c6beedb90f8b5a835580824d8b06fd1ff90c7c1
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Wed Mar 24 12:56:09 2021 +0100

    Reformat after rebase

commit c8ede87c45333f4c4f7acc1147fa482e6bca086a
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Mar 23 07:44:38 2021 +0100

    Add @covers annotation to phpunit tests

commit 4bf78b9a750085fc0d06dd87a7dc492a6a0c0ee8
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Mar 22 20:57:30 2021 +0100

    Use composer install in GA workflow

commit 070f9b26cc9f98dbec1a4b686e3e9913a44b2a11
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Mon Mar 22 08:46:56 2021 +0100

    Update composer dependencies

commit c8ea504760b4125789dd756eceb21a63fbdcfb64
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Fri Mar 19 14:47:01 2021 +0100

    Format generated config.php on Docker

commit ab8c83270e5ab7173b170ee3e5aa72131595fad7
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Fri Mar 19 14:44:14 2021 +0100

    Fix indentation issues

    In the change from two to fours spaces indentation, the following
    cases were not considered:
    - `switch` blocks.
    - Multiline control statements.
    - Multiline assignments/returns.
    - Multiline strings.
    - Semicolon after method chain calls.
    - Indentation after inline comments.

    Also, normalize multiline array's final bracket in new line to reduce
    diffs, except in templates.

commit bbcf8d3b250248eaa5a03478eb56fc5214ed2cef
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Wed Mar 17 04:56:32 2021 +0100

    Fix single line ifs and other template tweaks

    Single line ifs got braces added and indentation got broken. Use ternary
    operator or PHP tags on each line. Other minor fixes.

commit 741b64cc4af5f29f5f1ef070a6a03df8ba33a023
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sun Mar 14 04:35:21 2021 +0100

    Remove switch statement from template

commit 171fee6b7e2a7169808f866de4e499cfc69e8259
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Mar 16 21:36:51 2021 +0100

    Remove broken and mostly unneded template comments

commit 2d9183b55cbbc80284a454c0e869ee557dbd8236
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Mar 16 22:05:38 2021 +0100

    Update php-cs-fixer configuration

commit 7b504dd84090187483d22494150ea115efad7d12
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Wed Mar 17 05:10:15 2021 +0100

    indentation

commit 63bcf90f5929b04d78d0652104e1c77cd96f741e
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Mar 16 21:21:37 2021 +0100

    braces

    ['allow_single_line_anonymous_class_with_empty_body' => true,
    'allow_single_line_closure' => true]

commit c53763ec7b8c9f68f7390163639a852005bcfb28
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sun Mar 14 00:13:38 2021 +0100

    simple_to_complex_string_variable

commit a0b026432e323c990e338ef1ac43914f45e4b4f6
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sun Mar 14 00:10:31 2021 +0100

    return_assignment

commit c912975145dac53b34738e00d6c6a0f0d7c7ab31
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sun Mar 14 00:03:39 2021 +0100

    phpdoc_order

commit 0da40d3f775998984f865aabce0df607c4438f56
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sun Mar 14 00:01:41 2021 +0100

    phpdoc_no_empty_return

commit 9d99f560e5e6b6b46489c35f03dfa11f68fd4511
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 23:58:41 2021 +0100

    phpdoc_add_missing_param_annotation

commit 8fdaebe2506c2db588fdb7cd6f69e952bd273ac4
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:36:13 2021 +0100

    php_unit_test_class_requires_covers

commit d9b7e3a2bbee9ed41ffd6572837774f97011f612
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:33:58 2021 +0100

    php_unit_internal_class

commit ea9b017bd101519f58aec708ca90104b5f601667
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:32:38 2021 +0100

    operator_linebreak

    ['only_booleans' => true]

commit d7a552ef5fd5a1add41e0df20498ac96afeb5917
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:30:02 2021 +0100

    no_useless_return

commit ae926f2cd88cd52d077a4aecfd7396af55c545cf
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:27:58 2021 +0100

    no_useless_else

commit ff9525c3be7c96d172089f2280bfedc71c0d3a56
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:22:15 2021 +0100

    no_superfluous_elseif

commit 9981441077249a1358472cc9986de0e628b7c9d3
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:20:10 2021 +0100

    no_null_property_initialization

commit 494ac6b72c7ea4734138cb43ff5cc9adbe02bc04
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:18:39 2021 +0100

    multiline_whitespace_before_semicolons

commit 99b6371f05713870004d1e4aec6b52deca1aa82f
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:16:29 2021 +0100

    multiline_comment_opening_closing

commit 5ffc9c6a7ad0a82121209c1f7b44c0b2bd12f634
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:14:14 2021 +0100

    method_chaining_indentation

commit 6da8523ab0b07b36219162ac3942c90e04fe4b85
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:12:20 2021 +0100

    heredoc_to_nowdoc

commit f6f6bfc238be15cb03b923641745a0558b2cdbdd
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:10:12 2021 +0100

    explicit_string_variable

commit cf8a99491ede1626bb9057420ac82cfa583f5094
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:07:49 2021 +0100

    explicit_indirect_variable

commit 756a357cd590c7248c6e355907164ff89284d485
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:06:10 2021 +0100

    escape_implicit_backslashes

commit a5428fd1909cb0181187039a5e98dcc59c20cd86
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:03:16 2021 +0100

    combine_consecutive_unsets

commit cda4a478d068e7db4715e42fa83d234a4c090a72
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 22:01:36 2021 +0100

    combine_consecutive_issets

commit 45835e2f884fda7c70d383383073d1b71ab4200b
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 21:59:52 2021 +0100

    array_indentation

commit f87eecb2c25f6ecda18d8e8a01b5dd2b60e326ce
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 21:01:50 2021 +0100

    yoda_style

commit 80d79c75c9a1f710898540c8622e09386906ef1c
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:52:01 2021 +0100

    whitespace_after_comma_in_array

commit 36875df1543fdd8c009f0dd7c6ea96edb5aa8892
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:50:20 2021 +0100

    unary_operator_spaces

commit ce0e99125307eb22c5d09c19c51b3ee4a2e2a8bc
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:48:17 2021 +0100

    trim_array_spaces

commit 4d1904aa0da1bdececc2acfb377162bce4c88fd8
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:45:34 2021 +0100

    trailing_comma_in_multiline_array

commit e2c410662c5b2c458b2384974a04fac40d7ffe07
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:42:30 2021 +0100

    switch_continue_to_break

commit 6b72c4a8dfecb293baf675c3ffe7b769f15c8990
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:40:14 2021 +0100

    standardize_increment

commit 6666392643a61b3cb2011653b630baa060824bad
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:38:57 2021 +0100

    space_after_semicolon

    ['remove_in_empty_for_expressions' => true]

commit 7c89a36981d94b0ef7a58f863e9fb6bdfee525ad
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:35:25 2021 +0100

    single_space_after_construct

    ['constructs' => ['abstract', 'as', 'attribute', 'break', 'case',
    'catch', 'class', 'clone', 'comment', 'const', 'const_import',
    'continue', 'echo', 'elseif', 'extends', 'final', 'finally', 'for',
    'foreach', 'function', 'function_import', 'global', 'goto', 'if',
    'implements', 'include', 'include_once', 'instanceof',
    'insteadof', 'interface', 'match', 'named_argument', 'new',
    'open_tag_with_echo', 'php_doc', 'php_open', 'print', 'private',
    'protected', 'public', 'require', 'require_once', 'return', 'static',
    'throw', 'trait', 'use', 'use_lambda', 'use_trait', 'var', 'while',
    'yield', 'yield_from']]

commit 53f52cd12c3cda2fff55a59107de5f67fde67eb6
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:11:58 2021 +0100

    single_quote

commit d40f1b7d40ddc6d463d04e76858304a1093ae08d
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:06:46 2021 +0100

    single_line_comment_style

commit be8aa48add5edb943c069dd4ee3e63b94ffa6d9b
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 20:02:02 2021 +0100

    semicolon_after_instruction

commit 94881815e1d3851e62c978ac1771c22eeed4e653
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:57:21 2021 +0100

    phpdoc_types_order

commit 110c678ff71f04a00077c4a85d25fb18b8cbacd2
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:55:49 2021 +0100

    phpdoc_types

commit 7b0a4db399dc59055ad261e5042bcb571e261f1c
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:54:10 2021 +0100

    phpdoc_trim_consecutive_blank_line_separation

commit 44d9849c727c2a465be02b2689d6b703b519faf1
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:52:50 2021 +0100

    phpdoc_trim

commit 8aa55ccea78506a9898b42c3a97d37cde7aee4c8
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:51:50 2021 +0100

    phpdoc_to_comment

commit 9693b6e077b826afd60fe20cc2e39511b8f20633
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:49:55 2021 +0100

    phpdoc_summary

commit 1ec5795ab596ca8b55d01a3c9ab8e01edb849419
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:46:15 2021 +0100

    phpdoc_separation

commit 3de7dd552822f94740c68b4bc40e8176f83ce87f
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:44:49 2021 +0100

    phpdoc_scalar

commit b202b1b56a3019851fcd28dd555e4efcafcdde4d
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:42:09 2021 +0100

    phpdoc_no_package

commit 9a0875a922bd25dfa3e483ce2d9211b2b9144626
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:41:19 2021 +0100

    phpdoc_no_alias_tag

commit bc1ed87ba2447742b551772aa430c31662e67391
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:37:50 2021 +0100

    phpdoc_no_access

commit 46c2f55ed1ef2f2c4e6e3b69c1a678dc937987a6
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:35:10 2021 +0100

    phpdoc_indent

commit 191ecec93ff9a961bcce062ca545e78548be3528
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:33:35 2021 +0100

    phpdoc_annotation_without_dot

commit 91e7299ff2a3e223fbf2ce480ee757dae7089757
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:32:23 2021 +0100

    phpdoc_align

    ['tags' => ['method', 'param', 'property', 'return', 'throws',
    'type', 'var']]

commit c3d2cfada6e5d365f1a2af8c2b083b008bee1539
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:27:21 2021 +0100

    no_whitespace_before_comma_in_array

commit 09a9f2f99ba7a0fdabfd9b8a7b3b43468b20fc66
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:22:14 2021 +0100

    no_unneeded_control_parentheses

    ['statements' => ['break', 'clone', 'continue', 'echo_print',
    'return', 'switch_case', 'yield', 'yield_from']]

commit 4a9c2ba0b2e4e1ff143624b5326557c492a4d616
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:18:56 2021 +0100

    no_trailing_comma_in_singleline_array

commit 7949e2bba6fece5ea202f9e2e78c506e34c265f4
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:14:19 2021 +0100

    no_singleline_whitespace_before_semicolons

commit c70cd0a800df7b97cc1bee0c8480d7a5eb9981dc
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 19:11:45 2021 +0100

    no_multiline_whitespace_around_double_arrow

commit e1d80fff0ef8690cd827565fcd7f8baaa1d45c6a
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:58:44 2021 +0100

    no_mixed_echo_print

commit a109e738cd4016697f27794fe09faaaf8acba005
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:52:09 2021 +0100

    no_extra_blank_lines

    ['tokens' => ['break', 'case', 'continue', 'curly_brace_block',
    'default', 'extra', 'parenthesis_brace_block', 'return',
    'square_brace_block', 'switch', 'throw', 'use', 'use_trait']]

commit b12a13ceffd761aa95b763666aac1d89f8474292
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:48:52 2021 +0100

    no_empty_statement

commit 1e6464a808a5e573325998645d8f41f8e88aaaf5
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:46:57 2021 +0100

    no_empty_comment

commit 84c827aca61167dbf2d4f35f5d83cb2e90616c7a
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:45:01 2021 +0100

    no_blank_lines_after_phpdoc

commit 63a28449fb1af61da88be3a721950d149d7dc7e8
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:43:26 2021 +0100

    no_alternative_syntax

commit a813b0dec4d34d45fbead046f967124ca30781d8
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:40:54 2021 +0100

    no_alias_language_construct_call

commit 094991179724925ac21d18727c6fec31fe56283b
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:39:47 2021 +0100

    native_function_type_declaration_casing

commit 24f6b15eb7e4075c476507fca0a6560d95a602e1
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:38:52 2021 +0100

    native_function_casing

commit 736337e0bf775c1f22d051e23d73bb491246a788
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:35:12 2021 +0100

    increment_style

commit 69712547cf3234d0f7670b3a9af36fe2fb16f4dd
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:25:59 2021 +0100

    include

commit 969cd1091002673e0045c5f3f6ba2889173e1ae8
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:22:02 2021 +0100

    fully_qualified_strict_types

commit 4f056e0051a6aa32a2ab459d9264c6e3014c53d0
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:19:55 2021 +0100

    concat_space

commit 854d6e18b5b83385d829df3c6e033248c19ced81
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:16:33 2021 +0100

    class_attributes_separation

    ['elements' => ['method' => 'one']]

commit 23869ff5173ff3dd23727046db84bf959c9103f5
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:14:07 2021 +0100

    cast_spaces

commit 2801dd19f75a3a894f4ce090f4584cdd914b609a
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:12:27 2021 +0100

    blank_line_before_statement

    ['statements' => ['break', 'case', 'continue', 'declare',
    'default', 'exit', 'goto', 'include', 'include_once', 'require',
    'require_once', 'return', 'switch', 'throw', 'try']]

commit eb6eefa633d2189f4ae557de6ce23ec537232846
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:09:45 2021 +0100

    binary_operator_spaces

commit 929be1348de1722aab9067c4fbf9f608a295114b
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 18:06:15 2021 +0100

    array_syntax

    ['syntax' => 'short']

commit 6909f09525a58c51f76f86b63439ae2c0f044c61
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:59:21 2021 +0100

    ternary_operator_spaces

commit b0b75786f2eb52874b59aa11f0e191b5fe5b4e24
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:56:23 2021 +0100

    short_scalar_cast

commit 1f6519833c0d68bac967c5af52d4d74d8c0e2d05
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:53:56 2021 +0100

    return_type_declaration

commit f3e34d290791744d9f2af5bdcc6501b82cb8d626
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:50:55 2021 +0100

    ordered_class_elements

commit e2d6308ac1803f14345211c7ab7fad3cc15ae92b
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:48:26 2021 +0100

    no_whitespace_in_blank_line

commit 862c37d07f02b0e0e4088fdcc8021a7dfdeb911b
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:35:22 2021 +0100

    no_blank_lines_after_class_opening

commit 225f6e15717856e3e678efb059b486cd3b10ddab
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:22:22 2021 +0100

    new_with_braces

commit 863ff549190f1b90547de73bbeeadde190dce87b
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:18:18 2021 +0100

    blank_line_after_opening_tag

commit 721d5a958d5bf596424a8c58141ca3c203f6d0b9
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:12:34 2021 +0100

    visibility_required

    ['elements' => ['const', 'method', 'property']]

commit 282616f893ecddd462e1e074fa09d8423ee88c17
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:06:41 2021 +0100

    switch_case_semicolon_to_colon

commit feaec22508ddf1418e467a7d888c7c5c14a4539b
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:03:45 2021 +0100

    single_class_element_per_statement

commit 2e5b49d76b4d517dcb9f2c988040d9caa94178be
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 17:00:36 2021 +0100

    single_blank_line_at_eof

commit e1355833d0f42d3da7c5bcf75edadbaeeb9f18f1
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:59:28 2021 +0100

    no_trailing_whitespace_in_comment

commit 1b64e907ac1bae0cfd3cd964a3387e5fcd04f57f
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:57:32 2021 +0100

    no_trailing_whitespace

commit 8e449268b83f5ea354830ef0f0caf057e174b2dd
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:56:23 2021 +0100

    no_spaces_inside_parenthesis

commit 499dae27f70841a87d3789a3441e5f1f3302069f
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:54:55 2021 +0100

    no_spaces_after_function_name

commit 9645eb69ef04ccc2af69e7c4e5689ef673d13f15
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:53:37 2021 +0100

    no_closing_tag

commit 645a0ecf2349cb8ba70e73e4a821e0596faa0f13
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:50:44 2021 +0100

    no_break_comment

commit 5b546a0f4bc9c17d424afb467e6f9af484b49137
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:48:42 2021 +0100

    method_argument_space

commit d8b45d3103870a7733ae4ae5c7af430b9f00e47e
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:32:48 2021 +0100

    lowercase_keywords

commit d1f5ba26e1363c2745e278f8a3629894ce809d8c
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:31:13 2021 +0100

    line_ending

commit 15004d556a795d9bc0ee164873e0bd4a9606ce9b
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:28:56 2021 +0100

    indentation_type

commit 27e5bba47afcb362c1a54ced00ed6463dd9c6db3
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:24:23 2021 +0100

    function_declaration

commit 199ee9ada81359f8cfdc67c3ec272d7f107a6668
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:21:55 2021 +0100

    elseif

commit 55e8b4dc7bf3273a63c2b71f322c04fee956017e
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:20:41 2021 +0100

    constant_case

commit eeca8b91732286733666642b7195bab83adbe1db
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Sat Mar 13 16:15:47 2021 +0100

    class_definition

    ['single_line' => true]

commit 5af4b19148d65438e8959bde23f6bfb291e228e9
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Mar 9 01:31:14 2021 +0100

    WIP: Setup php-cs-fixer, refs #13317
